src/Config/CS_SDK.props

```diff
@@ -34,18 +33,37 @@
     <Optimize>true</Optimize>
     <DefineConstants>TRACE</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup Condition="$(Platform.Contains('NET60'))" >
+    <TargetFramework>net6.0</TargetFramework>
+    <!--Needed to copy nuget package assemblies to output folder. Anet6 issue-->
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
```

Comment:
so... does this turn all references into copy local true?

Reply:
yes, it copies all referenced assemblies in the output path

Reply:
that sounds dangerous - do we have duplicate copies now of things like newtonsoft.json in the `/nodes` and `/bin` folders?

Reply:
duplicate copies will not happen because all projects that have the outputPath to `/nodes` also have a redirect attribute
```
    <ReferenceCopyLocalPaths>
	  <!--Copy all assembly references to the OutputPath parent dir (/nodes/../) -->
	  <DestinationSubDirectory>..\</DestinationSubDirectory>
    </ReferenceCopyLocalPaths>
    ```